openmetadata-service/src/main/java/org/openmetadata/service/formatter/decorators/MSTeamsMessageDecorator.java

```diff
@@ -115,11 +116,26 @@ private TeamsMessage createMessage(
     String entityType = event.getEntityType();
 
     return switch (entityType) {
-      case Entity.TEST_CASE -> createDQMessage(publisherName, event, outgoingMessage);
+      case Entity.TEST_CASE -> createTestCaseMessage(publisherName, event, outgoingMessage);
       default -> createGeneralChangeEventMessage(publisherName, event, outgoingMessage);
     };
   }
 
+  private TeamsMessage createTestCaseMessage(
+      String publisherName, ChangeEvent event, OutgoingMessage outgoingMessage) {
+    final String testCaseResult = "testCaseResult";
```

Comment:
this should be at the class level, do not create final constants in method.

Reply:
Addressed.